Source/PLCrashAsyncMachExceptionInfo.c

```diff
@@ -29,7 +29,7 @@
 #include "PLCrashAsyncMachExceptionInfo.h"
 #include "PLCrashFeatureConfig.h"
 
-#if PLCRASH_FEATURE_MACH_EXCEPTIONS
```

Comment:
You need to update the define [here](https://github.com/microsoft/plcrashreporter/blob/b7b88ee14bbc25ce408ae05464cb6f1cdd747948/Source/PLCrashFeatureConfig.h#L86) instead of changing all usages